docs/services/aws-services-guide.md

```diff
@@ -815,76 +437,18 @@ becomes available, then navigate to the address. You should see the following:
 
 ![alt wordpress](wordpress-start.png)
 
-## Uninstall
-
-### Wordpress
-
-All Wordpress components that we installed can be deleted with one command:
-
-```bash
-kubectl delete -f wordpress-app.yaml
-```
-
-### Crossplane Configuration
+## Clean Up
 
-To delete all created resources, but leave Crossplane and the AWS stack
-running, execute the following commands:
+Because we put all of our configuration in a single directory, we can delete it all with this command:
 
 ```bash
-kubectl delete -f mysql-claim.yaml
-kubectl delete -f mysql-standard.yaml
-kubectl delete -f aws-mysql-standard.yaml
-kubectl delete -f app-project1-dev-namespace.yaml
-kubectl delete -f aws-infra-dev-namespace.yaml
+kubectl delete -f wordpress/
```

Comment:
Does this need to be `-Rf`?

Reply:
All files are top-level in `wordpress/` to no need to delete recursively.